src/gurt/telemetry.c

```diff
@@ -31,12 +31,17 @@ struct shmem_region_list {
 struct d_tm_shmem_hdr {
 	uint64_t		 sh_base_addr;	/** address of this struct */
 	key_t			 sh_key;	/** key to access region */
-	bool			 sh_deleted;	/** marked for deletion */
+	uint32_t		 sh_deleted:1,	/** marked for deletion */
+				 sh_require_ephemeral_lock:1; /** ephemeral_lock required */
```

Comment:
I am a little hesitant to have the flag be quite this specific. Maybe a name like "sh_multiple_writers" would better document our reasoning for needing a flag to begin with, and would allow us to do more things dependent on that, if we find more synchronization is needed later.

Reply:
yes, makes sense. Thanks